maestro-studio/web/src/pages/InteractPage.tsx

```diff
@@ -49,6 +49,7 @@ const InteractPage = () => {
         try {
           const deviceScreen = await API.getDeviceScreen();
           setDeviceScreen(deviceScreen);
+          await wait(250);
```

Comment:
Would it make more sense for this to be a minimum duration per iteration instead of a flat 250ms additional to every iteration? I guess it depends on what the hypothesis is here - can you explain the reasoning behind this?

Reply:
I have researched a bit more about adding this delay, I think it doesn't make sense here actually, since ```const deviceScreen = await API.getDeviceScreen();
          setDeviceScreen(deviceScreen);``` takes about 600ms at best to complete and for more complex screens it is >1s.